packages/wpcom.js/test/test.wpcom.site.post.subscriber.js

```diff
@@ -50,93 +50,87 @@ describe( 'wpcom.site.post.subscriber', function () {
 				} );
 			} );
 		} );
-	} );
 
-	describe( 'wpcom.site.post.subscriber.mine', function () {
-		it( 'should get subscribe status `FALSE` for current user', function () {
+		it( 'should get subscribers post list = 1', function () {
```

Comment:
The changes in this file are group existing `describe` blocks that had the same description, nothing has change in the tests. Is not easy to visualize in the diff :(